main.js

```diff
@@ -1,9 +1,17 @@
 const fs = require('fs');
+const path = require('path');
 
-const textExample1 = fs.readFileSync('testData/cat.html', 'utf-8');
-const textExample2 = fs.readFileSync('testData/dog.css', 'utf-8');
-const textExample3 = fs.readFileSync('testData/world.txt', 'utf-8');
-const textsForAnalysis = new Array(textExample1, textExample2, textExample3);
+const folderPath = 'testData';
+const pathFiles = fs.readdirSync(folderPath).map(fileName => {
+  return path.join(folderPath, fileName);
+});
```

Comment:
There is no need `{}` symbols and `return` statement in one line arrow function, so more short variant will be:

```
const pathFiles = fs.readdirSync(folderPath).map(fileName => path.join(folderPath, fileName););
```

Reply:
changes made